src/node/blockchain/service.go

```diff
@@ -15,7 +15,7 @@ import (
 )
 
 const (
-	DefaultPort = 8106
+	DefaultPort = 8080
```

Comment:
Actually we need two different ports:
* One for the node, the `DefaultPort` of this `service.go` (I choosen 8106)
* One for the UI server. We need to add a new constant called `DefaultPort` in `controller.go`. This constant should have the value 8080
In the "ruthenium\src\ui\main.go" file, the port program argument default value will be `server.DefaultPort` then.

Sorry if I was not clear enough in the issue description.


```suggestion
	DefaultPort = 8106
```

Reply:
I opened another [PR](https://github.com/my-cloud/ruthenium/pull/81) to fix this issue. If there is still no activity from your side at the end of the week, I will close this PR and merge mine.